### PR TITLE
One time Docs generation

### PR DESCRIPTION
**Pull request by aboobakersiddiqr63**
**Notes for Reviewers**

This PR fixes #

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ✓] Yes, I signed my commits.

### CODE REVIEW

File: docs/pages/integrations/apache-shardingsphere-operator-charts.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+list: include
+image: /assets/img/integrations/apache-shardingsphere-operator-charts.svg
+---
+
+<h1>{{ page.title }} with <span style="font-weight: bold; color: green;">Meshery</span> <img src="{{ page.image }}" style="width: 35px; height: 35px;" /></h1>
+
+
+#### Category: apache-shardingsphere-operator-charts
+
+### Overview & Features:
```
Comment (leecalcote) [inline docs/pages/integrations/apache-shardingsphere-operator-charts.md:18]: In a future revision, we might want to use some `{% include %}` macros.

File: docs/pages/integrations/apache-shardingsphere-operator-charts.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+display-title: "true"
+language: en
+list: include
+image: /assets/img/integrations/apache-shardingsphere-operator-charts.svg
+---
+
+<h1>{{ page.title }} with <span style="font-weight: bold; color: green;">Meshery</span> <img src="{{ page.image }}" style="width: 35px; height: 35px;" /></h1>
+
+
+#### Category: apache-shardingsphere-operator-charts
```
Comment (leecalcote) [inline docs/pages/integrations/apache-shardingsphere-operator-charts.md:16]: Uh-oh. This "category" is incorrect.

Comment (leecalcote) [reply docs/pages/integrations/apache-shardingsphere-operator-charts.md:16]: This is the "model".

File: docs/pages/integrations/apache-shardingsphere-operator-charts.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
+---
+layout: default
+title: Apache Shardingsphere Operator Charts
+permalink: integrations/apache-shardingsphere-operator-charts
+type: installation
+category: integrations
+display-title: "true"
```
Comment (leecalcote) [inline docs/pages/integrations/apache-shardingsphere-operator-charts.md:7]: We'll want to have new frontmatter values, like:

```
model: 
model-category:
model-subcategory: (in the future)
```

Comment (aboobakersiddiqr63) [reply docs/pages/integrations/apache-shardingsphere-operator-charts.md:7]: @leecalcote sure will work on it :)

Comment (aboobakersiddiqr63) [reply docs/pages/integrations/apache-shardingsphere-operator-charts.md:7]: @Lee do i need to change the Category to Model right ??

File: docs/pages/integrations/apache-shardingsphere-operator-charts.md (hunk 2 above)
Comment (aboobakersiddiqr63) [reply docs/pages/integrations/apache-shardingsphere-operator-charts.md:16]: @leecalcote so i need to change this category to model right 
The header should be Model

File: docs/pages/integrations/apache-shardingsphere-operator-charts.md (hunk 3 above)
Comment (leecalcote) [reply docs/pages/integrations/apache-shardingsphere-operator-charts.md:7]: Thank you. In the meantime.... the meshery.io site is pointing to "undefined" links. 😬

**Pull request merged**